episerver-cms-in-azure/azuredeploy.parameters.json

```diff
@@ -0,0 +1,36 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "WebApplicationWebAppName": {
+      "value": "GEN-UNIQUE"
+    },
+    "WebApplication-HostingPlanNameName": {
+      "value": "GEN-UNIQUE"
+    },
+    "WebApplication-HostingPlanNameSKU": {
+      "value": "Standard"
+    },
+    "WebApplication-HostingPlanNameWorkerSize": {
+      "value": "1"
+    },
+    "StorageAccountName": {
+      "value": "GEN-UNIQUE"
+    },
+    "sqlserverName": {
+      "value": "GEN-UNIQUE"
+    },
+    "sqlserverAdminLogin": {
+      "value": "GEN-USER"
```

Comment:
GEN-USER isn't an actual placeholder. For users, we usually do something generic like 'azureuser' or use GEN-UNIQUE